tools/functional/api/test_merge_requests.py

```diff
@@ -95,3 +98,116 @@ def test_merge_request_merge(project):
     with pytest.raises(gitlab.GitlabMRClosedError):
         # Two merge attempts should raise GitlabMRClosedError
         mr.merge()
+
+
+def merge_request_create_helper(
```

Comment:
This could be turned into a fixture factory in `conftest.py` (scoped to functions) that we can then use in any other test as well. The convention seems to be to create a `make_merge_request` fixture with an inner `_make_merge_request` function.
https://docs.pytest.org/en/6.2.x/fixture.html#factories-as-fixtures

Although not sure all of the logic should then stay in that fixture, maybe some asserts would belong in the test 🤔 

We have something similar here
https://github.com/python-gitlab/python-gitlab/blob/1508eb78a03b8d9429e474b7a6814ffe74517abb/tools/functional/conftest.py#L68-L78

Reply:
@nejch 

I have created a pytest.fixture for merge_request.

Let me know what you think! 😊